docs/trino-connector/installation.md

```diff
@@ -52,7 +52,7 @@ You can see the connector directory `gravitino-trino-connector-<version>` after
 Copy the connector directory to the Trino container's plugin directory.
 
 ```shell
-docker copy  /tmp/gravitino-trino-connector-<version> trino-gravitino:/lib/trino/plugin
+docker cp /tmp/gravitino-trino-connector-<version> trino-gravitino:/lib/trino/plugin
```

Comment:
@diqiu50 
Could you please help verify it?

Reply:
That‘s Ok